testapps/test_elasticsearch/tests/test_prod_es_indices.py

```diff
@@ -55,7 +55,7 @@ def test_prod_config(self):
     },
     {
         "alias": "hqusers",
-        "index": "test_hqusers_2017-02-03",
+        "index": "test_hqusers_2017-03-22",
```

Comment:
We need to the user index, to index a series of properties that werent in the mapping file. It is my understanding that that requires a reindex. Is this untrue? Either way, a user reindex should be pretty short.